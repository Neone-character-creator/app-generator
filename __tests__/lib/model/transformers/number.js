const numberTransformer = require("../../../../lib/model/transformers/number");

describe("number transformer", () => {
    it("transforms a string into a default object", () => {
        const config = "number";
        const transformed = numberTransformer(config);
        expect(transformed.type).toBe("number");
        expect(transformed.min).not.toBeDefined();
        expect(transformed.max).not.toBeDefined();
    });
    it("throws an error if value isn't number", () => {
       expect(()=>numberTransformer("string")).toThrowErrorMatchingSnapshot();
    });
});