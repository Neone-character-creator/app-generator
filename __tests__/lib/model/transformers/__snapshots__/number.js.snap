// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`number transformer throws an error if value isn't number 1`] = `"This transformer is for numbers but input is string"`;
